context/AuthContextProvider.tsx

```diff
@@ -87,7 +91,7 @@ export const AuthContextProvider = ({
    * @returns {Promise<void>}
    */
   const getUser = useCallback(async () => {
-    if (!isSessionInLocalStorage()) {
+    if (!isSessionInLocalStorage() && pathName !== '/register') {
       router.push('/login');
       return;
     }
```

Comment:
I'm confused by this. It's supposed to get the user but is just saying "If there is no session in local storage and the pathname does not equal `/register` push them to `/login`?

How is this getting the user?